app/src/main/java/org/oppia/app/player/state/StateAdapter.kt

```diff
@@ -121,4 +141,26 @@ class StateAdapter(
       binding.executePendingBindings()
     }
   }
+
+  inner class SelectionInteractionViewHolder(
+    val binding: ViewDataBinding
```

Comment:
Please make this binding private to minimize access to it.

Reply:
made it private.